### PR TITLE
Add new MagNet tool functions

### PR DESCRIPTION
**Pull request by elsevers**
Add functions to read:

- iron losses from a component and 
- parameters

The parameters function has been tested. Iron loss function has not been tested.

### CODE REVIEW

**Comment by elsevers** [comment]
This is ready to review and merge

**Review by ngadiyar93 — DISMISSED** [review]

File: tools/magnet/plumbing/document/solution/mn_ds_getIronLossInComponent.m [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+%   Returns taOhmicLoss
+%
+%
+%   Example: 
+
+if (exist('solutionID') ~= 1)
+    pid = 1;
+end
+
+invoke(MN, 'processcommand', 'ReDim Losses(1)');
```
Comment (ngadiyar93) [inline tools/magnet/plumbing/document/solution/mn_ds_getIronLossInComponent.m]: `MN` may please be replaced by lowercase `mn`

File: tools/magnet/plumbing/document/solution/mn_ds_getIronLossInComponent.m [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+invoke(mn, 'processcommand', ...
+    ['call getDocument().getSolution().getIronLossInComponent(' ...
+    num2str(pid) ', "' componentName '", Losses)']);
+
+invoke(mn, 'processcommand', 'Call setVariant(0, Losses, "MATLAB")');
+Losses = invoke(MN,  'getVariant', 0, 'MATLAB');
```
Comment (ngadiyar93) [inline tools/magnet/plumbing/document/solution/mn_ds_getIronLossInComponent.m]: `MN` may please be replaced by lowercase `mn`

File: tools/magnet/plumbing/document/solution/mn_ds_getIronLossInComponent.m (hunk 1 above)
Comment (elsevers) [reply tools/magnet/plumbing/document/solution/mn_ds_getIronLossInComponent.m]: Thanks

**Comment by elsevers** [comment]
This PR is ready for review + merging

**Review by ngadiyar93 — APPROVED** [review]
Looks good to me

**Review by horychen — APPROVED** [review]
Looks good to me.

**Pull request merged**